### ISSUE TITLE
NSDictionary cannot be converted to NSNumber

### ISSUE THREAD

**Issue by shinesunny** [leecade/react-native-swiper#213]
I am facing this issue while implementing showButtons={true}

JSON value '{
    animated = 1;
    x = 2048;
    y = 0;
}' of type NSDictionary cannot be converted to NSNumber

**Comment by getnashty** [comment]
Hi @shinesunny, any chance you can make a gif showing this behavior? I'm having a difficult time understanding the problem.

**Comment by shinesunny** [comment]
@getnashty 
![screen shot 2016-08-26 at 9 44 08 am](https://cloud.githubusercontent.com/assets/13848368/17993889/ba58e10a-6b71-11e6-9803-8717bf607191.png) I have issues with my quicktime player so I am uploading screen shot. Fast tapping on the buttons some time the index get incremented and then shows a view like this.

**Comment by shinesunny** [comment]
<Swiper loop={true} showsButtons={true}  bounces={false}
              dot={<View style={{backgroundColor:'rgba(0,0,0,.2)', width: 20, height: 20,borderRadius: 16, marginLeft: 3, marginRight: 3, marginTop: 3, marginBottom: 3,}} />}
              activeDot={<View style={{backgroundColor: 'red', width: 20, height: 20, borderRadius: 16, marginLeft: 3, marginRight: 3, marginTop: 3, marginBottom: 3,}} />}
              paginationStyle={{
                bottom: 323, left: null, right: 470,
              }}
                 nextButton={<Image source={require('../../assets/images/buttonright.png')} style={styles.rightArrow}/>} prevButton={<Image source={require('../../assets/images/buttonleft.png')} style={styles.leftArrow}/>}>
                <Image source={{uri: this.props.firstimage}}   style={styles.image}/>
                <Image  source={{uri: this.props.secondimage}} style={styles.image}/>
                <Image  source={{uri: this.props.thirdfimage}} style={styles.image}/>
            </Swiper>

**Comment by getnashty** [comment]
2 questions,

1. did you create that view with the ActivityIndicator?
2. are you using the loadMinimal property?

**Comment by shinesunny** [comment]
Yes I used ActivityIndicatorIOS. But I didn't using loadMinimal property.

**Comment by getnashty** [comment]
Ok, (side note, ActivityIndicatorIOS will be deprecated at some point) - how can we know what the problem is, if it looks like your view is rendering? Is that view meant to change through some prop?

**Comment by shinesunny** [comment]
The view rendering and index increments.

**Comment by shanavasHassan** [comment]
@getnashty how can I control transition speed on button tap?

**Comment by getnashty** [comment]
I'm away this week - but I will definitely take a look at this when I'm
back next week. I don't believe it's modifiable as of now, but probably
would be simple to add

Sent from my iPhone

On Aug 28, 2016, at 7:26 PM, shanavasHassan <notifications@github.com>
wrote:

@getnashty <https://github.com/getnashty> how can I control transition
speed on button tap?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/leecade/react-native-swiper/issues/213#issuecomment-243034878>,
or mute the thread
